pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -620,17 +620,21 @@ func (p *MasqueradePodInterface) preparePodNetworkInterfaces() error {
 			return err
 		}
 	}
-	if Handler.IsIpv6Enabled() && (Handler.HasNatIptables(iptables.ProtocolIPv6) || Handler.NftablesLoad("ipv6-nat") == nil) {
-		err = Handler.ConfigureIpv6Forwarding()
-		if err != nil {
-			log.Log.Reason(err).Errorf("failed to turn on net.ipv6.conf.all.forwarding")
-			return err
-		}
+	if Handler.IsIpv6Enabled() {
+		if Handler.HasNatIptables(iptables.ProtocolIPv6) || Handler.NftablesLoad("ipv6-nat") == nil {
+			err = Handler.ConfigureIpv6Forwarding()
+			if err != nil {
+				log.Log.Reason(err).Errorf("failed to turn on net.ipv6.conf.all.forwarding")
```

Comment:
I know this is not the changed part but consider it as you already touched it:
- This level does not  (supposed) to know what `net.ipv6.conf.all.forwarding` is, that is one layer down knowledge. So it can be added there into the returned error.
- The caller most likely already logs the error, so it makes sense to just concatenate the current error with whatever info is needed from this level and return using `fmt.Errorf` without logging anything here.

Reply:
Changed the error to not contain implementation details.